spec/query_spec.cr

```diff
@@ -17,8 +17,10 @@ describe LuckyRecord::Query do
 
   describe "#first" do
     it "gets the first row from the database" do
-      insert_a_user
-      UserQuery.new.first.name.should eq "Paul"
+      insert_a_user("First")
+      insert_a_user("Last")
```

Comment:
You can now use Boxes. If you have time it would be awesome to switch all these `insert_a` methods to use the boxes instead in a separate PR. Here is an example:

https://github.com/luckyframework/lucky_record/blob/4c2abf7e4251e66f25fb9e91f1b82bead8463444/spec/associations_spec.cr#L6

Reply:
I added an issue here https://github.com/luckyframework/lucky_record/issues/157

Reply:
Sweet. I'll address in a separate PR